build/README.md

```diff
@@ -358,3 +359,12 @@ instead of `make shell` to start an interactive shell for development on Minikub
 #### `make minikube-transfer-image`
 Convenience target for transferring images into minikube.
 Use TAG to specify the image to transfer into minikube
+
+## Built With
+
+- [healthcheck](https://github.com/heptiolabs/healthcheck)
+- [errors](https://github.com/pkg/errors)
+- [pflag](https://github.com/spf13/pflag)
+- [viper](https://github.com/spf13/viper)
+- [grpc](https://godoc.org/google.golang.org/grpc)
+- [context](https://golang.org/pkg/context/)
```

Comment:
I feel like context is a core library in go, so it's probably not worth listing here.

Do we want to just list everything in here:
https://github.com/GoogleCloudPlatform/agones/blob/master/Gopkg.toml

(Or conversely, just point people to it?)

Reply:
@enocom wdyt?

Reply:
A link for people unaware of dep's convention of using `Gopkg.toml` might be nice.

Reply:
I don't have strong opinions on this one.

Maybe something like "This project uses `dep` (with link) as a dependency manager. You can see the list of dependencies here (link)"

That's probably the most valuable without having to (a) rewrite docs for another project (b) keep a list up to date in two places.

WDYT?

Reply:
SGTM!